src/skillmodels/kalman_filters.py

```diff
@@ -177,9 +182,6 @@ def kalman_predict(
             the sigma_point algorithm chosen.
         sigma_weights (jax.numpy.array): 1d array of length n_sigma with non-negative
             sigma weights.
-        transition_info (dict): Dict with the entries "func" (the actual transition
```

Comment:
Was the docstring of `transition_info` outdated or did you actually change this? If you changed it, why is the mapping from columns to positions no longer needed?

Reply:
I did not change any functionality, but did check that the only thing ever used was the function. No idea when this stopped being required, for sure had been the case [2.5 years ago](https://github.com/OpenSourceEconomics/skillmodels/commit/22fb915a894ba12dd8a90822680ec380dcadc1a4#diff-f67fef4e2f70a1271df3182635d6e146f32efbe1518e83b7211f36d0d3422975).

Reply:
*(so yes, I did actually change from passing a dict to passing the function only)*

Reply:
Maybe this was before we went full dags? But if it wasn't used you can of course remove it.